From 3e2269aa9189ea50ef4d7688a2e0297604267351 Mon Sep 17 00:00:00 2001
From: Liu Ying <b17645@freescale.com>
Date: Wed, 18 Nov 2009 08:52:02 -0500
Subject: [PATCH 133/505] ENGR00118328 MXC V4l2:Fix build break for several
 unit test cases

1)Remove unusable headfiles included in unit test code.
2)Include mxcfb.h headfile for csi_v4l2_capture.c.

Signed-off-by: Liu Ying <b17645@freescale.com>
---
 test/mxc_v4l2_test/csi_v4l2_capture.c | 1 +
 test/mxc_v4l2_test/mxc_v4l2_capture.c | 1 -
 test/mxc_v4l2_test/mxc_v4l2_overlay.c | 1 -
 test/mxc_v4l2_test/mxc_v4l2_still.c   | 1 -
 test/mxc_v4l2_test/mxc_v4l2_tvin.c    | 1 -
 5 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/test/mxc_v4l2_test/csi_v4l2_capture.c b/test/mxc_v4l2_test/csi_v4l2_capture.c
index b602a33..c0fb9fc 100644
--- a/test/mxc_v4l2_test/csi_v4l2_capture.c
+++ b/test/mxc_v4l2_test/csi_v4l2_capture.c
@@ -38,6 +38,7 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
+#include <linux/fb.h>
 #include <linux/videodev.h>
 #include <sys/mman.h>
 #include <sys/time.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_capture.c b/test/mxc_v4l2_test/mxc_v4l2_capture.c
index 98252cd..56ac8ef 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_capture.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_capture.c
@@ -38,7 +38,6 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
-#include <linux/compiler.h>
 #include <linux/videodev.h>
 #include <sys/mman.h>
 #include <string.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_overlay.c b/test/mxc_v4l2_test/mxc_v4l2_overlay.c
index 5e53775..d9d592f 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_overlay.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_overlay.c
@@ -39,7 +39,6 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
-#include <linux/compiler.h>
 #include <linux/videodev.h>
 #include <linux/fb.h>
 #include <sys/mman.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_still.c b/test/mxc_v4l2_test/mxc_v4l2_still.c
index 9bd36b6..4b11e19 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_still.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_still.c
@@ -35,7 +35,6 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
-#include <linux/compiler.h>
 #include <linux/videodev.h>
 #include <sys/mman.h>
 #include <string.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_tvin.c b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
index 66d8492..f2cc005 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_tvin.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
@@ -39,7 +39,6 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
-#include <linux/compiler.h>
 #include <linux/videodev.h>
 #include <linux/videodev2.h>
 #include <linux/fb.h>
-- 
1.8.0

